requirements.txt

```diff
@@ -4,7 +4,7 @@ oauth2client<4.0.0
 pandas>=0.22.0,<1.0.0
 six>=1.11.0,<2.0.0
 tensorflow==1.13.0rc1; python_version=='2.7'
-tensorflow-transform==0.11.0; python_version=='2.7'
-tensorflow-data-validation==0.11.0; python_version=='2.7'
-tensorflow_metadata==0.9.0
+tensorflow-transform==0.12.0; python_version=='2.7'
+tensorflow-data-validation==0.12.0; python_version=='2.7'
+tensorflow_metadata==0.12.0
```

Comment:
Weirdly, there is no such 0.12.0, but there is a 0.12.1.

BUT in their `setup.py` the `__version__` is still 0.12.0. 

I recommend bumping this to 0.12.1 or doing a bound.

Reply:
I made a comment for them here: https://github.com/tensorflow/metadata/issues/8#issuecomment-466517442